flask_dance/contrib/authentiq.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import unicode_literals
+
+from functools import partial
+
+from flask.globals import LocalProxy, _lookup_app_object
+
```

Comment:
Any particular reason for all the blank lines here? It's just a problem, but it looks a bit weird, especially since none of the other configurations are like this.